arrows/pytorch/CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+###
+# Loadable algorithms
+
+set(kwiver_python_subdir "python${PYTHON_VERSION}${PYTHON_ABIFLAGS}")
+set(kwiver_python_output_path "${KWIVER_BINARY_DIR}/lib/${kwiver_python_subdir}")
+
+kwiver_create_python_plugin_init(kwiver/arrows/pytorch)
+
+# TODO: change KWIVER_ENABLE_PYTHON to KWIVER_ENABLE_PYTORCH
```

Comment:
Do this

Reply:
Added in https://github.com/Kitware/kwiver/pull/691/commits/53e9542e400dc8939d54f9170fd6e1117d42fd10